### PR TITLE
Launching Other Programs from Python

### PR DESCRIPTION
**Pull request by sanjibansg**
### Your checklist for this pull request
:warning: Please review the [guidelines for contributing](https://github.com/sanscript-tech/hacking-tools-scripts/blob/main/CONTRIBUTING.md) to this repository.

Parent issue: #49 

### Type of change

**Select all that applies**:
* [X ] New feature (non-breaking change which adds functionality)
* [ ] Bug fix (non-breaking change which fixes an issue)
* [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
* [ ] This change requires a documentation update

### Checklist:

* [X] My code follows the style guidelines(Clean Code) of this project.
* [X] I have performed a self-review of my own code.
* [X] I have used consistent code conventions and clear function/method/variable names.
* [X] I have commented on my code, particularly in hard-to-understand areas.
* [X] I have created a helpful and easy to understand README.md which contains description, requirements and use case.
* [ ] I have added relevent screenshots/images if necessary in a seperate folder.
* [ ] I have added my work in respective language folder.
* [X] My changes generate no new warnings.

### CODE REVIEW

File: Launching other Programs/main.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#Python script to launch anyother program
+import os
+import subprocess
+
```
Comment (yashaswibiyahut) [inline Launching other Programs/main.py:4]: Kindly add some choice to users like calculator, notepad, Wordpad.

File: Launching other Programs/README.md [hunk 2]
```diff
@@ -0,0 +1 @@
+Folder contains Python Script to launch any other program in Windows Operating System.
```
Comment (yashaswibiyahut) [inline Launching other Programs/README.md:1]: Add a proper readme file with an example/use case.

File: Launching other Programs/main.py [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+program="calculator path"
```
Comment (yashaswibiyahut) [inline Launching other Programs/main.py]: Please add an exception or error

**Review by yashaswibiyahut — CHANGES_REQUESTED** [review]

File: Launching other Programs/main.py (hunk 1 above)
Comment (sanjibansg) [reply Launching other Programs/main.py:4]: Choices for the other programs should be given using user input or argument variables or predefined choices in the script? @yashaswibiyahut

Comment (yashaswibiyahut) [reply Launching other Programs/main.py:4]: user input

**Review by yashaswibiyahut — CHANGES_REQUESTED** [review]

**Review by yashaswibiyahut — APPROVED** [review]

**Pull request merged**